apps/paymaster-proxy/src/rpc/getJsonRpcRequestHandler.ts

```diff
@@ -0,0 +1,57 @@
+import type * as Express from 'express'
+
+import { processSingleOrMultiple } from '@/helpers/processSingleOrMultiple'
+import type { SponsorUserOperationImpl } from '@/paymaster/types'
+import { validateJsonRpcRequest } from '@/rpc/validateJsonRpcRequest'
+import { wrapPaymasterResponseIntoJsonRpcResponse } from '@/rpc/wrapPaymasterResponseIntoJsonRpcResponse'
+import type { SupportedPaymasterRequest } from '@/schemas/supportedPaymasterRequestSchema'
+
+const handlePaymasterMethod = async <
+  T extends SupportedPaymasterRequest = SupportedPaymasterRequest,
+>(
+  request: T,
+  sponsorUserOperation: SponsorUserOperationImpl,
+) => {
+  switch (request.method) {
```

Comment:
it might be clear for this to become if/else statements if any more logic gets added here, generally think its a good idea to avoid a switch statement if you don't need the default

i also wonder if it's possible to have a map of some sort that basically maps method name to handler

Reply:
will do this in a subsequent PR once we have more than one method